protocol/messenger_handler.go

```diff
@@ -1321,6 +1321,8 @@ func (m *Messenger) HandleSyncPairInstallation(state *ReceivedMessageState, mess
 	// TODO(samyoul) remove storing of an updated reference pointer?
 	state.AllInstallations.Store(message.InstallationId, installation)
 	state.ModifiedInstallations.Store(message.InstallationId, true)
+	targeted := message.TargetInstallationId == m.installationID
+	state.ModifiedInstallationsTargetedToThisDevice.Store(message.InstallationId, targeted)
```

Comment:
Can we only store the `InstallationId` and not `targeted`. The name of the property makes it seem like it would only contain targeted installations already.